components/dialog/test/dialog-mixin.vdiff.js

```diff
@@ -51,6 +76,24 @@ describe('dialog-mixin', () => {
 					});
 				});
 			});
+
+			it('delayed-content', async() => {
+				const el = await fixture(
+					`<d2l-dialog title-text="Delayed Dialog"><${delayedTag}></${delayedTag}></d2l-dialog>`,
+					{
+						awaitLoadingComplete: false
```

Comment:
This is a special configuration option because otherwise the fixture call itself would wait for all the descendants to be loading-complete. In this case, we want to bypass that since we want to make sure that dialog itself waits for them.